Sources/ArcGISToolkit/Utility/CredentialInputView.swift

```diff
@@ -210,65 +210,58 @@ struct CredentialInputSheetView: View {
     }
     
     var body: some View {
-        GeometryReader { proxy in
-            VStack {
-                VStack(alignment: .center) {
-                    VStack(spacing: 8) {
-                        Text(title)
-                            .font(.title)
-                            .multilineTextAlignment(.center)
-                        Text(message)
-                            .font(.subheadline)
-                            .foregroundColor(.secondary)
-                            .multilineTextAlignment(.center)
-                    }
-                    .padding(.vertical)
-                    VStack {
-                        switch fields {
-                        case .password:
-                            passwordTextField
-                        case .usernamePassword:
-                            usernameTextField
-                            Divider()
-                            passwordTextField
-                        }
-                        Divider()
-                    }
-                    .padding([.bottom, .horizontal])
-                    HStack {
-                        Spacer()
-                        Button(role: .cancel) {
-                            cancelAction.handler("", "")
-                        } label: {
-                            Text(cancelAction.title)
-                                .padding(.horizontal)
+        NavigationStack {
+            GeometryReader { proxy in
+                VStack {
+                    VStack(alignment: .center) {
+                        VStack(spacing: 8) {
+                            Text(title)
+                                .font(.title)
+                                .multilineTextAlignment(.center)
+                            Text(message)
+                                .font(.subheadline)
+                                .foregroundColor(.secondary)
+                                .multilineTextAlignment(.center)
                         }
-                        .buttonStyle(.bordered)
-                        Spacer()
-                        Button {
-                            isPresented.wrappedValue = false
-                            continueAction.handler(username, password)
-                        } label: {
-                            Text(continueAction.title)
-                                .padding(.horizontal)
+                        .padding(.vertical)
+                        Form {
+                            switch fields {
+                            case .password:
+                                passwordTextField
+                            case .usernamePassword:
+                                usernameTextField
+                                passwordTextField
+                            }
                         }
```

Comment:
```suggestion
                        }
                        .scrollContentBackground(.hidden)
```

The background difference between the header is a bit jarring, thoughts?

| Before | After |
|---|---|
| <img src="https://github.com/user-attachments/assets/89d5eb1e-cd6d-487d-90a2-2ad2b0af40c5"> | <img src="https://github.com/user-attachments/assets/db0c8269-9a2e-4bbe-aee4-f9ce43be0916"> |

Reply:
I noticed that too but I believe I left it there because of some default behavior on visionOS, let me check.

Reply:
Nope looks good.